### PR TITLE
Fixed the bug where a candidate from previous phase could not re-apply

### PR DESCRIPTION
**Pull request by imnotteixeira**
Fixed by removing the `unique` from `Candidato`'s `email` and adding a check in the application to see if that email already exists in the same phase.

As a consequence, the index must be removed from the mongo collection as well, by doing:
```
db.candidatos.dropIndex("email_1")
```

Also fixed eslint, because it was giving me problems with indentation :/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,8 +22,10 @@
   },
   "devDependencies": {
     "chai": "^4.1.2",
-    "eslint": "^3.19.0",
+    "eslint": "^6.1.0",
+    "eslint-config-airbnb-base": "^14.0.0",
```
Comment (miguelpduarte) [inline package.json]: Why this? I don't see it being used anywhere 😅

File: routes/views/candidatura.js [hunk 2]
```diff
@@ -1,72 +1,75 @@
...
 
-			novaCand.save(function (err) {
-				if (err) {
-					if (err.name === 'MongoError' && err.code === 11000) {
-						req.flash('error', 'Apenas te podes candidatar uma vez!');
-
-						res.redirect('/candidatura');
+			Candidato.model.findOne({
+				email: req.body.email,
+				fase_candidatura: req.body.fase_candidatura,
```
Comment (miguelpduarte) [inline routes/views/candidatura.js]: 👍

**Comment by miguelpduarte** [comment]
@imnotteixeira You must have forgotten to stage the changes before amending, the force push shows no diff.

**Comment by imnotteixeira** [comment]
> @imnotteixeira You must have forgotten to stage the changes before amending, the force push shows no diff

Fixed

**Pull request merged**